app/controllers/admin/edition_images_controller.rb

```diff
@@ -26,10 +26,15 @@ def create
     @new_image = @edition.images.build
     @new_image.build_image_data(image_params["image_data"])
 
+    if @new_image.image_data
+      @new_image.image_data.validate_on_edition = @edition
+      @new_image.image_data.validate_on_image = @new_image
+    end
```

Comment:
Are there any circumstances where this `if` statement _won't_ be true? 🤔 

Considering we `build_image_data` in the line above, I'm thinking this will _always_ be present. But maybe I'm missing an edge case?

Reply:
I think you're right, I was thinking we'd need this if we had an empty file upload for this scenario but the image data still exists in this case.